boards/st/nucleo_g431kb/doc/index.rst

```diff
@@ -0,0 +1,189 @@
+.. _nucleo_g431kb_board:
+
+ST Nucleo G431KB
+################
+
+Overview
+********
+
+The Nucleo G431kB board features an ARM Cortex-M4 based STM32G431kB MCU
+with a wide range of connectivity support and configurations.
+Here are some highlights of the Nucleo G431kB board:
+
+- STM32 microcontroller in LQFP32 package
+- Arduino nano V3 connectivity
+- On-board ST-LINK/V3E debugger/programmer
+- Flexible board power supply:
+
+  - USB VBUS or external source(3.3V, 5V, 7 - 12V)
+  - Power management access point
+
+- Three LEDs: USB communication (LD1), power LED (LD3), user LED (LD2)
+- One push-button for RESET
+
+.. image:: img/nucleo_g431kb.webp
+  :align: center
+  :alt: Nucleo G431kB
+
+More information about the board can be found at the `Nucleo G431KB website`_.
+
+- Development support: serial wire debug (SWD), JTAG, Embedded Trace Macrocell*
```

Comment:
Assuming this is a copy-paste from somewhere, the asterisk probably doesn't make sense here?


```suggestion
- Development support: serial wire debug (SWD), JTAG, Embedded Trace Macrocell.
```